### PR TITLE
chore(ui): add bluesky button

### PR DESCRIPTION
**Pull request by a2937**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Related to #54572 

I had to update fontAwesome to support BlueSky. 

I'm not sure where `ShareTemplateWrapperTestID` is used and have only tested if the content loads. I will be double-checking both if the buttons display correctly and the new button can be used to share on BlueSky later today.

### CODE REVIEW

**Comment by a2937** [comment]
Share cert


This looks pretty well. I haven't made a test message yet, but I believe it will work if I read the docs right.

https://docs.bsky.app/docs/advanced-guides/intent-links#compose-post

**Comment by ilenia-magoni** [comment]
Will you also add Instagram and Linkedin?

**Comment by a2937** [comment]
Wait.  I don't actually know how to add LinkedIn buttons for clearing a challenge; nor how they're supposed to appear on the profile page.

**Comment by a2937** [comment]
I've made a Share on Instagram button for the lower jaw when a challenge is completed. Hopefully it works well.

File: client/i18n/locales/english/translations.json [hunk 1]
```diff
@@ -106,7 +106,9 @@
     "donate-now": "Donate Now",
     "confirm-amount": "Confirm amount",
     "play-scene": "Press Play",
-    "closed-caption": "Closed caption"
+    "closed-caption": "Closed caption",
+    "share-on-bluesky": "Share on BlueSky",
+    "share-on-insta": "Share on Threads"
```
Comment (naomi-lgbt) [inline client/i18n/locales/english/translations.json:111]:
```suggestion
    "share-on-threads": "Share on Threads"
```

This'll take more changes than just this line, but we shouldn't refer to Threads as Insta(gram) anywhere in the code, or that'll just make a mess when we get to the point where we *want* to include Instagram links.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Comment by ilenia-magoni** [comment]
there are merge conflicts

**Review by naomi-lgbt — APPROVED** [review]

**Review by Sembauke — APPROVED** [review]

**Pull request merged**